packages/components/src/components/video/video.ts

```diff
@@ -0,0 +1,86 @@
+import { css, html } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { HasSlotController } from '../../internal/slot';
+import { property } from 'lit/decorators.js';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Used to wrap external video elements (e. g. from Moving Image or bare <video>-Tags) and provide some basic styling for Union Investment.
+ * @status experimental
```

Comment:
pls change before merging

Reply:
You mean the `status`?

Reply:
what should status be?

Reply:
status and version.

Status should be stable and version needs to be updated to the latest +1 right before merging.